utils/getInvoices/tests/getInvoices.placing.test.ts

```diff
@@ -5,139 +5,892 @@ import { expect } from '@jest/globals'
+      )
+    })
+
+    it('should NOT load when service = null, company = null, prevPayment = { invoice: [Inflicion] }', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = null
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Placing,
+            price: 10,
+            sum: 110,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
 
-    const invoices = getInvoices({
-      company,
-      service,
-      payment,
-      prevPayment,
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
     })
+    
+    it('should NOT load when service = null, company = null, prevPayment = { invoice: [Cleaning] }', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = null
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Cleaning,
+            price: 10,
+            sum: 110,
+          },
+        ],
+      }
 
-    expect(invoices).toContainEqual({
-      type: ServiceType.Placing,
-      price: payment.invoice[0].sum,
-      sum: payment.invoice[0].sum,
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
+    })
+
+    it('should NOT load when service = { rentPrice: 10 }, company = null, prevPayment = { invoice: [Cleaning] }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = null
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Cleaning,
+            price: 10,
+            sum: 110,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
+    })
+
+    it('should NOT load when service = { rentPrice: 10 }, company = null, prevPayment = { invoice: [Inflicion] }', () => {
+      const service: Partial<IService> = {
+        rentPrice: 10,
+      }
+      const company: Partial<IRealestate> = null
+      const prevPayment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Placing,
+            price: 10,
+            sum: 110,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevPayment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
+    })
```

Comment:
not required in current stage
better to use such tests after we properly load data from `prevPayment = { invoice: [Placing] }`

Reply:
> better to use such tests after we properly load data from `prevPayment = { invoice: [Placing] }`

Just like you does as line 550